editors/js/editionVE.js

```diff
@@ -3507,7 +3507,7 @@ mergeLine: function (direction,cbError) {
           ednVE.sclEd.synchSelection();
         }
       }
-      e.stopImmediatePropagation();
+      // e.stopImmediatePropagation();
```

Comment:
Instead of commenting out this line, we could make the ednVE to allow registering other dblclick event handlers on graphemes from other places. In that way, from our plugin, we could explicitly register the handler with the ednVE.

Reply:
The architecture is setup with events that refer to the action not the mapped gesture.  This is on purpose so that different VE's are decoupled and can determine the appropriate mapping of user gestures to actions. The editionVE broadcasts a message to all open editors informing that element selection has changed. Any editor simply writes a handle if appropriate.

Reply:
@xadxura the pull request is commenting out this code. We would have to test the editionVE fully to understand the behavior.  Changing the editionVE handling of dblclick is not an option for this version, as this is a known gesture response. @yangli0516 the editionVE responds dblclick events depending on mode specific to edition editting and viewwing.  This is encapsulated behavior.  Can you define how you would like to change this behavior so that we can discuss it. Otherwise, perhaps just responding to the raised event is sufficient?